Onboarding/Tests/Mocks/MockAccessibilityPoster.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import UIKit
+
+@testable import Onboarding
+
+struct MockAccessibilityPoster: AccessibilityPoster {
+    static var _postParams: (UIAccessibility.Notification, Any?)? = nil
```

Comment:
`_receivedPostNotification` & `_receivedPostArgument` please. Tuples can get confusing really quickly.

Reply:
Any objection to a new struct to replace the tuple? Makes more sense in the tests where I need to verify if the function was called or not as I can test if the struct is nil rather than individual vars?

Reply:
I would rather individual vars if testing the vars themselves... it matches elsewhere.
If you are simply testing if something is called have a `var _postCalled: Bool = false` that you set to true if its called.

Reply:
Done